vagrant/ansible/playbook.yml

```diff
@@ -0,0 +1,156 @@
+---
+- hosts: all
+
+  handlers:
+    - name: restart postgres
+      service: name=postgresql state=restarted
+
+  tasks:
+  
+  - name: Install python2.7
+    raw: "sudo apt-get update -qq && sudo apt-get install -qq python2.7 aptitude"
```

Comment:
Why using a raw command instead of `package`?

The `package` declaration is independent of a particular package manager: http://docs.ansible.com/ansible/latest/package_module.html#package-generic-os-package-manager